build.gradle

```diff
@@ -30,6 +30,7 @@ subprojects {
       opentelemetryJavaagent     : "1.23.0-SNAPSHOT",
       opentelemetryJavaagentAlpha: "1.23.0-alpha-SNAPSHOT",
 
+      jackson                    : "2.14.2",
```

Comment:
We should never add a non-opentelemetry dependency if we can avoid it, _especially_ if it is such a common one like Jackson. This is effectively a guarantee for hard-to-debug classloading nightmares.